velox/common/base/Counters.h

```diff
@@ -128,4 +119,28 @@ constexpr folly::StringPiece kMetricSpillWriteTimeMs{
 
 constexpr folly::StringPiece kMetricFileWriterEarlyFlushedRawBytes{
     "velox.file_writer_early_flushed_raw_bytes"};
+
+constexpr folly::StringPiece kMetricArbitratorNumRequests{
+    "velox.arbitrator_num_requests"};
+
+constexpr folly::StringPiece kMetricArbitratorNumAborted{
+    "velox.arbitrator_num_aborted"};
+
+constexpr folly::StringPiece kMetricArbitratorNumFailures{
+    "velox.arbitrator_num_failures"};
+
+constexpr folly::StringPiece kMetricArbitratorQueueTimeUs{
+    "velox.arbitrator_queue_time_us"};
+
+constexpr folly::StringPiece kMetricArbitratorArbitrationTimeUs{
+    "velox.arbitrator_arbitration_time_us"};
+
+constexpr folly::StringPiece kMetricArbitratorNumShrunkBytes{
```

Comment:
Let's skip reporting the following three for now?

Reply:
We should keep the migration the same as presto-native and modify the migrated ones in followups if needed